src/app.jsx

```diff
@@ -416,6 +416,11 @@ const NavigatorCardBody = ({
     const isMounted = useRef(null);
 
     function calculateBoxPerRow () {
+        if (!isGrid) {
+            setBoxPerRow(1);
+            return;
+        }
```

Comment:
This makes the previous item scroll out of view that is not really what I expect to happen. All of this code has to be refactored soon for `react-window` support as well. So this might not be the best issue to work on at the moment

Reply:
You mean like here #212 ?
I was working on it btw but didn't reach to a complete code that can be pushed so I pushed this one first